tests/unit/hook_packages/terraform/hooks/prepare/prepare_base.py

```diff
@@ -1022,9 +1023,10 @@ def setUp(self) -> None:
                 f"AwsApigatewayv2RouteMyApigwv2Route{self.mock_logical_id_hash}": self.expected_cfn_apigwv2_route,
                 f"AwsApigatewayv2StageMyApigwv2Stage{self.mock_logical_id_hash}": self.expected_cfn_apigwv2_stage,
                 f"AwsApigatewayv2IntegrationMyApigwv2Integration{self.mock_logical_id_hash}": self.expected_cfn_apigwv2_integration,
+                f"AwsApigatewayv2AuthorizerMyAuthorizerV2{self.mock_logical_id_hash}": self.expected_cfn_apigwv2_authorizer,
             },
         }
-
+        self.maxDiff = None
```

Comment:
Why do we need to set this?

Reply:
Whoops, left over from debugging, will remove